addon/ng2/blueprints/ng2/files/__path__/vendor.ts

```diff
@@ -0,0 +1,17 @@
+// Typescript emit helpers polyfill
+import 'ts-helpers';
+
+import '@angular/core';
```

Comment:
why are we using the side-effect-y version of there in a vendor.ts file? this forces inclusion of a bunch of code somebody may never use?